src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/ControlFlowInstructions.cs

```diff
@@ -767,12 +767,12 @@ private ThrowInstruction(bool hasResult, bool isRethrow)
 
         public override int Run(InterpretedFrame frame)
         {
-            var ex = (Exception)frame.Pop();
+            var ex = frame.Pop();
```

Comment:
I think that according to [the corefx coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), `var` should not be used here now that it's not obvious what the variable type is.